mne/viz/evoked.py

```diff
@@ -473,10 +473,17 @@ def _plot_lines(data, info, picks, fig, axes, spatial_colors, unit, units,
                                 linewidth=0.5)[0])
                     line_list[-1].set_pickradius(3.)
 
-            if gfp:  # 'only' or boolean True
+            if gfp:
+                if gfp in [True, 'only']:
+                    this_gfp = D.std(axis=0, ddof=0)
+                    label = 'GFP'
+                elif gfp in ['power', 'power-only']:
+                    # RMS
+                    this_gfp = np.sqrt((D ** 2).mean(axis=0))
```

Comment:
```suggestion
                    this_gfp = np.linalg.norm(D, axis=0)
```
it avoids allocating D**2

Reply:
Nice one!!!!